api/src/main/java/io/strimzi/api/kafka/model/status/KafkaMirrorMaker2Status.java

```diff
@@ -33,6 +33,8 @@ public class KafkaMirrorMaker2Status extends KafkaConnectStatus {
 
     private List<Map<String, Object>> connectors = new ArrayList<>(3);
 
+    private List<AutoRestartStatus> autoRestartStatuses = new ArrayList<>();
```

Comment:
Does this need to use some list with clear sorting? Or is it not needed in this place? (if the fields are generated in random order, they will change order and will trigger constant upgrades of the CR in Kubernetes)

Reply:
I missed that part, I will implement clear sorting.